src/controllers/requestController.js

```diff
@@ -0,0 +1,26 @@
+import RequestService from '../services/requestService';
+
+/**
+ * Class for Requests
+ */
+export default class RequestController {
+/**
+ * Update a pending trip request
+ * @param {*} req - requset object
+ * @param {*} res - response object
+ * @return {json} - json
+ */
+  static async updateRequest(req, res) {
+    const data = req.body;
+    const { id } = req.params;
+    try {
+      if (data.tripType !== 'return') data.returnDate = null;
+
+      const result = await RequestService.updateRequest(Number(id), data);
+
+      res.status(200).json({ status: 'success', data: result });
```

Comment:
check the util folder, there is a helper function for this `res.status(200).json({ status: 'success', data: result });`, please ultilize that and adjust all your code